client/my-sites/site-settings/analytics/form-google-analytics.jsx

```diff
@@ -88,8 +90,9 @@ export const GoogleAnalyticsForm = ( props ) => {
 		placeholderText,
 		recordSupportLinkClick,
 		setDisplayForm,
+		isFreeWPCOM,
 	};
-	if ( props.siteIsJetpack ) {
+	if ( props.siteIsJetpack && ! isFreeWPCOM ) {
```

Comment:
I wonder whether in a future change we should adapt the `hasSiteAnalyticsFeature` selector to return something like `wpcom` and `jetpack`, so that we can keep this logic contained in the selector. 

Eventually that logic could also be pushed server-side, extending the plan features work @retrofox has been doing.

Reply:
That would definitely be an improvement. I've also wondered if we should continue to have two different Google Analytics form components.